data/systemd/snapd.service.in

```diff
@@ -17,6 +17,7 @@ OnFailure=snapd.failure.service
 OOMScoreAdjust=-900
 ExecStart=@libexecdir@/snapd/snapd
 EnvironmentFile=-@SNAPD_ENVIRONMENT_FILE@
+EnvironmentFile=-/var/lib/snapd/environment/snapd.conf
```

Comment:
Per discussion on MM, can you also add this to snapd.apparmor, snapd.failure, snapd.autoimport and snapd.snap-repair?

Reply:
This is done now